### PR TITLE
Restore "overlay" (borders) functionality

### PR DESCRIPTION
**Pull request by Apaczer**
This brings back "borders" feature to this port thus triggering overlay image over unscaled video screen with ``border.png`` put inside ./gpsp directory (set ``Display scalling->scaled 3:2``).

* Overlay sampel:
border

* Package:
[borders.zip](https://github.com/rik-smeets/gpsp/files/8995656/borders.zip)

### CODE REVIEW

**Comment by rik-smeets** [comment]
Thanks for doing this! I think showing a border should be an option for the user. Could you add a setting for this?

File: video.c [hunk 1]
```diff
@@ -4290,7 +4290,26 @@ void video_resolution_small()
...
+      SDL_BlitSurface(png, NULL, rl_screen, NULL);
+      SDL_Flip(rl_screen);
+      SDL_BlitSurface(png, NULL, rl_screen, NULL);
+      SDL_Flip(rl_screen);
+      SDL_FreeSurface(png);
+    }
+    break;
+  default:
+    break;
+  }
```
Comment (rik-smeets) [inline video.c:4306]: Instead of a switch statement, I think this can just be an if statement.

**Comment by Apaczer** [comment]
> I think showing a border should be an option for the user.

Agree but it might take some time for me to apply, could you point me in some dirrections? I'm in process of learning C. Anyway it's working as it is, the best solution would be swapping different overlays inside menu with any *.png files.

File: video.c (hunk 1 above)
Comment (Apaczer) [reply video.c:4306]: ok, should be more clean now.

**Comment by daFourch** [comment]
Nice one! May I suggest to also add "color correction", used in gpSP retroarch's core? It greatly helps reducing colors' contrast, as it used to be on the original hardware.

Keep up the good work!

**Pull request closed without merging**